designs/administrator/future-openshift/cluster-settings/index.md

```diff
@@ -0,0 +1,64 @@
+---
+parent: Adminstrator
+version: 4.6
+---
+
+# Cluster Settings 
+
+Users complete upgrades in cluster settings. The following design focuses on making improvements to the current upgrade experience in cluster settings by making upgrading as simple and transparent as possible. Specific improvements include adding more information around recommended upgrade paths and notifying users when new versions and channels become available. 
+
+## Adding Alerts to the Notification Drawer
+
+![Alerts - Available Updates](img/notifications-updates.png)
+- Today, available updates appear in the message section of the notification drawer. 
+- Three new alerts will be added to inform users if (1) a new patch becomes available, (2) a new minor release becomes available, and/or (3) new channels become available. 
+- All three messages send users to the cluster settings section of the console to take further action.
+
+## Channel Visualization and In Context Help
+
+The new channel visualization provides users with more transparency into recommended upgrade paths, informs users what changes happen in each release, and guides users to the correct channel. 
+
+In addition to the visualization: 
+- Field level help text was added for channels
+- A new manage subscription field was added with a direct link to manage subscriptions in OpenShift Cluster Manager
+
+![Channel help text](img/channel-help.png)
+- Clicking `Channel` surfaces a popover that provides context as to what a channel is along  with a link to documentation to learn more about channels in OpenShift.
+
+![Channel visualization](img/channel-visualization.png)
+- The blue line signifies the current channel you are on. The current version and recommended version (if available) are both marked with blue dots. 
+- The gray line signifies the next available channel.
+- To surface recommended versions in the next available channel, users must switch to the channel by clicking the `Pencil` next to `Channel`. 
```

Comment:
Not really integral to this design but it sounds like theres some possibility the pencil icon #421 will remain on the value vs label for 4.6.

Reply:
Updated the mockups to align with what we have today - pencil icon remaining on the value.